metricbeat/module/elasticsearch/shard/data.go

```diff
@@ -11,17 +11,19 @@ import (
 
 var (
 	schema = s.Schema{
-		"state":   c.Str("state"),
-		"primary": c.Bool("primary"),
-		"node":    c.Str("node"),
-		"index":   c.Str("index"),
-		"shard":   c.Int("number"),
+		"state":           c.Str("state"),
+		"primary":         c.Bool("primary"),
+		"node":            c.Str("node"),
+		"index":           c.Str("index"),
+		"shard":           c.Int("shard"),
+		"relocating_node": c.Str("relocating_node"),
```

Comment:
@pickypg This value is most of the time `null`. In these cases metricbeat just skips the value instead of sending `null`. Is that ok?

Reply:
Yes, that should be fine.